VMT/VMT.md

```diff
@@ -0,0 +1,225 @@
+
+The first steps performed by the VMT are to confirm the validity of the report, create a private GitHub Issue if necessary, prefix the description with an [embargo reminder](templates/reception-embargo-reminder.md), create a Kata Containers Security Announcement ([KCSA](templates/KCSA.md)) Issue, and subscribe the project's core security review team for confirmation of impact and determination of affected branches.
+Reports starting with an *Incomplete* KCSA Issue should have a corresponding [incomplete reception message](templates/reception-incomplete-message.md) added in a comment.
+Once the VMT confirms a KCSA is warranted, the KCSA Issue status will be set to *Confirmed*.
+If the need for a KCSA is challenged, the KCSA Issue status should be set back to *Incomplete* until that question is resolved.
+
+> **FIXME** Do we need to create a Kata Containers Security Team (KCST) (that is, a team of security experts in the Kata Containers community), along with the VMT (the key difference being the VMT drive the process, whereas the KCST are security experts and engineers who can make technical decisions and contributions).
+
+> **FIXME** We also need to define where we store and publish KCSN's - I expect on a GitHub md file in the appropriate repo?
+
+For some lower-risk issues or problems which may only be easy to solve in future releases, the KCSA Issue will be set to *Opinion* and the core security reviewers for the Kata Containers Security Team (KCST) will be subscribed to determine whether they wish to issue a Kata Containers Security Note (KCSN) (these reports may still sometimes remain under embargo until the KCSN is issued).
+If no KCSA is warranted and there is no benefit to a [KCSN](https://wiki.openstack.org/wiki/Security_Notes) then the KCSA Issue will be set to *Won't Fix* or *Invalid* (depending on the specific situation) and the bug state switched from *Private Security* to *Public*, optionally adding the *security* bug tag if the report concerns a potential security hardening opportunity.
+The specifics are indexed in the [report taxonomy](#incident-report-taxonomy) and [task status](#KCSA-task-status) tables.
+
+## Patch development
+
+For a private report, the reporter and the affected projects' core security review teams, plus anyone they deem necessary to develop and validate a fix, are added to the Issues subscription list.
+A fix is proposed as a patch to the current master branch (as well as any affected supported branches) and attached to the private Issue, **not sent to the public code review system**.
+
+For public reports, there is no need to directly subscribe anyone and patches can be submitted directly to the code review system instead as Pull Requests (PR).
+
+> **FIXME** - do we have a process for reaching out by the VMT or beyond the VMT - I would guess the only 'next step up' is going to be the Kata architecture committee, who right now are the VMT members :-) Maybe there is a route up to the OSF ?
```

Comment:
I think myself and (I'll volunteer him...) @fungi would be your OSF staff to bring in on something